sjb/config/common/test_cases/origin_built_installed_release_39.yml

```diff
@@ -58,6 +58,7 @@ extensions:
                          --connection local         \
                          --inventory sjb/inventory/ \
                          -e openshift_deployment_type=origin  \
+                         -e openshift_image_tag="$( cat ./ORIGIN_RELEASE )" \
```

Comment:
Does it make sense to update `sjb/config/common/test_cases/origin_built_installed_release.yml` as well?

Reply:
We haven't seen this happening on master yet, so I'd rather not touch working parts.